spec/core_functions/color/blackness.hrx

```diff
@@ -14,6 +14,17 @@ a {
   b: 100%;
 }
 
+<===> max/warning
+DEPRECATION WARNING: Individual color-channel functions defined globally or in the color module are deprecated in favor of the new color.channel() function.
```

Comment:
@stacyk One formatting issue I'm noticing right way is that the text is going over the character limit. I think Sass typically prefers line limit of 80 characters. (The `===` that separate each virtual file is 80 characters wide)

![bild](https://user-images.githubusercontent.com/41588129/201735646-c0361a2b-5af5-4ca7-b41c-8daf6daa624b.png)

Reply:
ah, yes. My editor was wrapping even though I didn't add the line break so I didn't catch this. Thanks @dvdherron